File: A15function/A13PracticeQuestion/A17DecToBin.java
package A15function.A13PracticeQuestion;

public class A17DecToBin {
    public static int DecToBin(int decNum) {
        int pow = 0;
        int binNum = 0;

        while (decNum > 0) {
            int rem = decNum % 2;
            binNum = binNum + rem * (int) Math.pow(10, pow);
            pow++;
            decNum = decNum / 2;
        }
        return binNum;
    }

    public static void main(String args[]) {
        System.out.println(DecToBin(27));
    }
}

File: A15function/A13PracticeQuestion/A17DecToBin.js
function decToBin(dec = 30) {
  let pow = 0;
  let binNum = 0;

  while (dec > 0) {
    let rem = dec % 2;
    binNum = binNum + rem * Math.pow(10, pow);
    dec = Math.floor(dec / 2);
    pow++;
  }
  console.log(binNum);
}

decToBin();
